packages/salesforcedx-vscode-core/src/commands/forceStartApexDebugLogging.ts

```diff
@@ -120,7 +120,7 @@ export class ForceStartApexDebugLoggingExecutor extends SfdxCommandletExecutor<{
 
 export async function getUserId(projectPath: string): Promise<string> {
   const defaultUsernameOrAlias = await getDefaultUsernameOrAlias();
-  if (isNullOrUndefined(defaultUsernameOrAlias)) {
+  if (defaultUsernameOrAlias === undefined) {
```

Comment:
It seems like `if (!defaultUsernameOrAlias)` should be here.  this way, if null or undefined the code will execute.  Or or we not want this block to run if null?

Reply:
A similar condition in `packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts` got changed to `!defaultUsernameOrAlias`.